psh/mkdir/mkdir.c

```diff
@@ -35,8 +36,9 @@ int psh_mkdir(int argc, char **argv)
 	}
 
 	for (i = 1; i < argc; i++) {
-		if (mkdir(argv[i], 0) < 0)
-			fprintf(stderr, "mkdir: failed to create %s directory\n", argv[i]);
+		if (mkdir(argv[i], 0) < 0) {
+			fprintf(stderr, "mkdir: failed to create %s directory (%s)\n", argv[i], strerror(errno));
```

Comment:
Nitpicking: I'll go for `"mkdir: failed to create %s directory: %s\n"` to maintain the same format as `perror` and GNU tools:
<img width="423" alt="image" src="https://user-images.githubusercontent.com/3939983/165817475-6efed2c7-96c6-4df9-ba43-afc0b28f8342.png">